src/coprocessor/dag/rpn_expr/types/mod.rs

```diff
@@ -56,4 +59,16 @@ impl<'a> RpnFnCallPayload<'a> {
     pub fn output_rows(&'a self) -> usize {
         self.output_rows
     }
+
+    /// Gets the length of implicit arguments
+    #[inline]
+    pub fn implicit_args_len(&'a self) -> usize {
+        self.implicit_args.len()
+    }
+
+    /// Get implicit argument at the special position
+    #[inline]
+    pub fn imp_param_at(&'a self, position: usize) -> &'a ScalarArg {
```

Comment:
Change the name to be consistent with `implicit_args_len`above

Reply:
Sorry. forgot it.